apps/volontulo/views/offers.py

```diff
@@ -526,3 +527,23 @@ def get(request):
         return render(request, 'offers/archived.html', {
             'offers': Offer.objects.get_archived()
         })
+
+
+class OffersGalleryDelete(DeleteView):
+    """Class based view to delete offers image."""
+    model = OfferImage
+
+    def get(self, request, *args, **kwargs):
```

Comment:
But finally it will remove object without confirmation when user go to URL. This should be done via POST.